data_handling/munge_ansirh.py

```diff
@@ -0,0 +1,124 @@
+"""
+Munge ANSIRH data into JSON files binned by state.
+
+Author(s): Kate Habich
+"""
+
+import pandas as pd
+import json
+from .clean_ansirh import clean_ansirh
+
+
+def main_ansirh():
+    """
+    Creates state dictionary of data from ANSIRH.
+
+    Returns (None):
+        Writes JSON file with cleaned and formatted ANSIRH data.
+    """
+
+    ansirh_data = pd.read_csv("./data/AFD_2021_for_ArcGIS_Upload.csv")
+
+    # drop empty column
+    ansirh_data = ansirh_data.drop(["Unnamed: 2"], axis=1)
+
+    # make dictionaries of every row in dataset
+    row_dicts = make_row_dicts(ansirh_data)
+
+    # clean data
```

Comment:
To cut down on comments, you could change this comment to be "# Clean and sort data" and then remove the comments (and line breaks) on lines 30-31 and 33-34. Also, for consistency, capitalize the first letter of each comment.